### ISSUE TITLE
[BUG] YAMLParseError in statistics

### ISSUE THREAD

**Issue by phlind** [st3v3nmw/obsidian-spaced-repetition#324]
**Describe the bug**
The statistics are not shown. Instead, I get this (for example):

```
YAMLParseError: Tabs are not allowed as indentation at line 2, column 1:

	type: bar
	labels: [0,1,2,3,4]
^
```

**To Reproduce**
Not sure how to reproduce it. It just shows in my statistics panel.

**Expected behavior**
Showing the statistics in graph form.

**Versions (please complete the following information):**
 - OS: macOS 11.4
 - Obsidian version: 0.13.5
 - Plugin version: 1.6.2

**Comment by andrewcrook** [comment]
Yes I am getting this aswell 
not sure if it’s the chart plugin or this plugin at fault

**Comment by dummifiedme** [comment]
I am getting this error as well.

PS. Is the owner of this project inactive? :frowning_face:

**Comment by st3v3nmw** [comment]
This has been fixed (#351), fix will be available after I'm done with tests.

**Comment by st3v3nmw** [comment]
Should be fixed in v1.7.0
